package.json

```diff
@@ -42,34 +42,34 @@
         "node": ">=16.10.0"
     },
     "peerDependencies": {
-        "typescript": "5.6.2"
+        "typescript": "5.6.3"
     },
     "dependencies": {
         "@typescript-to-lua/language-extensions": "1.19.0",
-        "enhanced-resolve": "5.8.2",
-        "picomatch": "^2.3.1",
-        "resolve": "^1.15.1",
-        "source-map": "^0.7.3"
+        "enhanced-resolve": "5.17.1",
+        "picomatch": "^4.0.2",
+        "resolve": "^1.22.8",
+        "source-map": "^0.7.4"
     },
     "devDependencies": {
-        "@types/fs-extra": "^8.1.0",
-        "@types/glob": "^7.1.1",
-        "@types/jest": "^27.5.2",
-        "@types/node": "^13.7.7",
-        "@types/picomatch": "^2.3.0",
-        "@types/resolve": "1.14.0",
-        "eslint": "^9.11.0",
-        "eslint-plugin-jest": "^28.8.3",
-        "fs-extra": "^8.1.0",
-        "javascript-stringify": "^2.0.1",
-        "jest": "^29.5.0",
-        "jest-circus": "^29.5.0",
-        "lua-types": "^2.13.0",
+        "@types/fs-extra": "^11.0.4",
+        "@types/glob": "^8.1.0",
+        "@types/jest": "^29.5.14",
+        "@types/node": "^22.9.0",
+        "@types/picomatch": "^3.0.1",
+        "@types/resolve": "1.20.6",
+        "eslint": "^9.14.0",
+        "eslint-plugin-jest": "^28.9.0",
+        "fs-extra": "^11.2.0",
+        "javascript-stringify": "^2.1.0",
+        "jest": "^29.7.0",
+        "jest-circus": "^29.7.0",
+        "lua-types": "^2.13.1",
         "lua-wasm-bindings": "^0.3.1",
-        "prettier": "^2.8.8",
-        "ts-jest": "^29.1.0",
-        "ts-node": "^10.9.1",
-        "typescript": "^5.6.2",
-        "typescript-eslint": "^8.7.0"
+        "prettier": "^3.3.3",
+        "ts-jest": "^29.2.5",
+        "ts-node": "^10.9.2",
+        "typescript": "5.6.3",
```

Comment:
I removed the caret here, since having it here was a bug. (This should be kept exactly the same as what the peer dependency specifies.)